### PR TITLE
ST7735 plugin

### PR DESCRIPTION
**Pull request by ananthb**
Copied over the ST7789 plugin for the ST7735.

### CODE REVIEW

File: pidi-display-st7735/setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ],
+    packages=["pidi_display_st7735"],
+    install_requires=[
+        "pidi-display-pil>=0.1.0",
+        "st7735",
+        "Pillow",
+    ],
+    entry_points={
+        'pidi.plugin.display': [
+            'DisplayST7735 = pidi_display_st7789:DisplayST7735'
```
Comment (Gadgetoid) [inline pidi-display-st7735/setup.py]: Should be `pidi_display_st7735:DisplayST7735`

Comment (ananthb) [reply pidi-display-st7735/setup.py]: Ah yeah

**Review by Gadgetoid — APPROVED** [review]

**Comment by Gadgetoid** [comment]
Thank you- how do you want to go about publishing this to pypi? Do you want to do the honors, or should I?

The base plugin for image based display output - pidi-display-pil - probably needs updating to support display resolutions other than 240x240. Also have some vague plans to support hiding the text/transport controls.

**Comment by ananthb** [comment]
Thank you for offering, but please go ahead. I'd like this one to exist under the same maintainer as the other pidi plugins.

I'd also like to tackle the pidi-display-pil work. I've got an ST7735 display with me already. Let me see how pidi looks on it.

**Comment by Gadgetoid** [comment]
> I'd also like to tackle the pidi-display-pil work. I've got an ST7735 display with me already. Let me see how pidi looks on it.

Happy for you to give it a shot!

And uh... sorry in advance for the horror of pidi-display-pil. It's weirdly put together to try and avoid too much heavy lifting/refreshing/slowdown.

**Pull request merged**